plugins/weltolk_backup_qq/weltolk_backup_qq_setting.php

```diff
@@ -0,0 +1,1626 @@
+<?php if (!defined('SYSTEM_ROOT')) {
+    die('Insufficient Permissions');
+}
+if (ROLE !== 'admin') {
+    msg('权限不足!');
+    die;
+}
+
+$limit = option::get('weltolk_backup_qq_limit');
+$enable = option::get('weltolk_backup_qq_enable');
+$log = option::get('weltolk_backup_qq_log');
+
+global $i, $m;
+$page = $_GET['page'];
```

Comment:
```suggestion
$page = isset($_GET['page']) ? $_GET['page'] : '';
```

Reply:
```suggestion
$page = $_GET['page'] ?? '';
```
https://stackoverflow.com/questions/3210935/whats-the-difference-between-isset-and-array-key-exists